app/services/cads/live_data_service.rb

```diff
@@ -25,11 +27,13 @@ def power_for_type(powers, type)
     end
 
     def api
-      unless @api
-        @api = MeterReadingsFeeds::GeoApi.new(username: ENV['GEO_API_USERNAME'], password: ENV['GEO_API_PASSWORD'])
-        @api.login
+      @api ||= MeterReadingsFeeds::GeoApi.new(token: token)
+    end
+
+    def token
+      Rails.cache.fetch(CACHE_KEY, expires_in: 45.minutes) do
```

Comment:
Can we make the cache timeout an environment variable here, so its easier to change if they change their API credentials.